CircuitPython_WeatherCloud/code.py

```diff
@@ -20,25 +21,29 @@
 button = digitalio.DigitalInOut(board.A1)
 button.switch_to_input(pull=digitalio.Pull.UP)
 
-wave_file = open("sound/Rain.wav", "rb")
-wave = audiocore.WaveFile(wave_file)
+with open("sound/Rain.wav", "rb") as wave_file:
```

Comment:
these changes are from linting

Reply:
Will do. Everything else in this good to go?

Reply:
You were right @FoamyGuy. Updated here: 1082a37124e77d7ea5668f20ccb5bd8b7f5101dd